python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -69,17 +70,36 @@ class StandardAutoscaler:
     until the cluster size that can handle the resource demand is met).
     """
 
-    def __init__(self,
-                 config_path,
-                 load_metrics,
-                 max_launch_batch=AUTOSCALER_MAX_LAUNCH_BATCH,
-                 max_concurrent_launches=AUTOSCALER_MAX_CONCURRENT_LAUNCHES,
-                 max_failures=AUTOSCALER_MAX_NUM_FAILURES,
-                 process_runner=subprocess,
-                 update_interval_s=AUTOSCALER_UPDATE_INTERVAL_S,
-                 prefix_cluster_info=False,
-                 event_summarizer=None,
-                 prom_metrics=None):
+    def __init__(
+            self,
+            config_path: str,
+            load_metrics: LoadMetrics,
+            max_launch_batch: int = AUTOSCALER_MAX_LAUNCH_BATCH,
+            max_concurrent_launches: int = AUTOSCALER_MAX_CONCURRENT_LAUNCHES,
+            max_failures: int = AUTOSCALER_MAX_NUM_FAILURES,
+            process_runner: Any = subprocess,
+            update_interval_s: int = AUTOSCALER_UPDATE_INTERVAL_S,
+            prefix_cluster_info: bool = False,
```

Comment:
@DmitriGekhtman `self.prefix_cluster_info` does not seem to be used

Reply:
It's used in the Kubernetes operator to prepend cluster identifiers for grepping log output.
https://github.com/ray-project/ray/blob/efed07023fdb824653f320325bc30f1b93ec6086/python/ray/ray_operator/operator.py#L114